faker/providers/company/en_PH/__init__.py

```diff
@@ -0,0 +1,178 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from collections import OrderedDict
+from .. import Provider as CompanyProvider
+
+
+class Provider(CompanyProvider):
+    """
+    Provider for company names for en_PH locale
+
+    Company naming scheme and probabilities are inspired by and/or based on existing companies in the Philippines.
+    Parody names of popular local companies are also available.
```

Comment:
Jokes are notoriously misinterpreted across cultures. I would drop the parody names.

Reply:
Well, that is why it is locale-specific hahaha! But sure, I will drop those.